cmd/vclusterctl/cmd/get/service_cidr.go

```diff
@@ -0,0 +1,78 @@
+package get
+
+import (
+	"fmt"
+
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/flags"
+	"github.com/loft-sh/vcluster/cmd/vclusterctl/log"
+	"github.com/loft-sh/vcluster/pkg/helm/values"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type serviceCIDRCmd struct {
+	Context   string
+	Namespace string
+	log       log.Logger
+}
+
+func getServiceCIDR(globalFlags *flags.GlobalFlags) *cobra.Command {
+	cmd := &serviceCIDRCmd{
+		Context:   globalFlags.Context,
+		Namespace: globalFlags.Namespace,
```

Comment:
Please use the global flags here like in the other commands:
```
cmd := & serviceCIDRCmd{
		GlobalFlags: globalFlags,
		log:         log.GetInstance(),
	}
```

Reply:
done